Package.swift

```diff
@@ -252,10 +252,10 @@ let package = Package(
             name: "WinUIInterop",
             dependencies: []
         ),
-        // .target(
-        //     name: "CursesBackend",
-        //     dependencies: ["SwiftCrossUI", "TermKit"]
-        // ),
+         .target(
+             name: "TermKitBackend",
+             dependencies: ["SwiftCrossUI", .product(name: "TermKit", package: "TermKit")]
```

Comment:
Use `"TermKit"` spelling instead of `.product(name: "TermKit", package: "TermKit")`